pyrdfstore/store.py

```diff
@@ -207,11 +253,19 @@ def select(self, sparql: str, named_graph: Optional[str] = None) -> Result:
         log.debug(f"Result from SPARQLStore :: {type(result)=} -> {result=}")
         return result
 
-    def insert(self, graph: Graph, named_graph: Optional[str] = NIL_NS):
+    def insert(
+        self,
+        graph: Graph,
+        named_graph: Optional[str] = NIL_NS,
+        safe_characters: Optional[str] = None,
+    ):
         graph = reparse(graph)
+        graph = clean_graph(graph, safe_characters)
```

Comment:
this possibly growing list of calls for cleaners to be called is precisely what urged me into thinking about a more flexible and extensible platform --> see #49

Reply:
also, decide to push this into the abstract-base-class, so implementations (subclasses) can rely on self.clean(graph)